src/app/service/bounty/RefreshBounty.ts

```diff
@@ -55,7 +55,9 @@ export default async (guildMember: GuildMember, bountyId: string, message: Messa
 		await message.edit(embedMessage);
 		addCompletedReactions(message);
 		break;
+	case 'Draft':
```

Comment:
@amaredeus, is this switch statement meant to carry through like this, or is there supposed to be a function for `draft`?

Reply:
Good question! The logic for cases "Draft", "Deleted", and default should all be the same. In this scenario if a bounty is refreshed for any of these cases then the message should be removed from the bounty board. Reason is that drafts should not be public, neither should be deleted, and if for some reason the status is null or undefined, delete the message too.